recipes/default.rb

```diff
@@ -139,7 +143,7 @@
   end
 
   package %w[php72w php72w-cli php72w-common php72w-curl php72w-gd php72w-mbstring
-             php72w-process php72w-snmp net-snmp ImageMagick jwhois nmap mtr 
+             php72w-process php72w-snmp net-snmp ImageMagick jwhois nmap mtr
```

Comment:
Is this cha'ge related to the feature you want to implement?

Reply:
This is not a change I meant to do. I think maybe my editor say the extra space and removed it. Sorry about that.